src/report_tests.py

```diff
@@ -14,7 +14,9 @@ def __init__(self, artifact_path: str) -> None:
         self.text = ''
 
     def append(self, status: str, test: str, comment: str = '') -> None:
-        link = Markdown.link(test, self.artifact_path + test)
+        name = test[test.rfind('/') + 1:test.rfind('.log')]
+        name = name.replace('_', ' ')
+        link = HTML.link(name, self.artifact_path + test)
```

Comment:
I'd extend the function to take `label` as an additional argument and make potential demangling the responsibility of the caller.